Dockerfile

```diff
@@ -159,11 +159,14 @@ COPY --from=builder /usr/src/target/release/text-generation-router /usr/local/bi
 # Install launcher
 COPY --from=builder /usr/src/target/release/text-generation-launcher /usr/local/bin/text-generation-launcher
 
+RUN apt update && apt install build-essential g++ -y
+
 # AWS Sagemaker compatbile image
 FROM base as sagemaker
 
 COPY sagemaker-entrypoint.sh entrypoint.sh
 RUN chmod +x entrypoint.sh
+RUN pip install triton
```

Comment:
This should be installed earlier: it will only be available to the sagemaker target if you install it here.
Maybe it is not required? If I remember correctly, pytorch now also installs it but maybe I'm wrong.

Reply:
Doesn't seem to be the case.
https://github.com/pytorch/pytorch/blob/main/requirements.txt

I know I had to install it somewhere, are you ok with adding directly in the requirements ? (I did that for now, will revert if needed)